### ISSUE TITLE
No overload matches

### ISSUE THREAD

**Issue by NightNutSky** [microsoft/vscode#176145]
- VS Code Version: 1.76.0
- OS Version: Windows 10

Something contradicts each other. Although, everything was ok in the past.

![](https://user-images.githubusercontent.com/70456337/222932568-f7d945af-5e68-46a1-b2c0-3eacf7dcf2b0.png)

![](https://user-images.githubusercontent.com/70456337/222932569-7cf2c4c5-643a-47db-aa4c-3c32fa9677a8.png)

**Comment by NightNutSky** [comment]
Also, should be noted. Kinda the same thing works in one part of code, and doesn't in the current part.

Here it is.
![image-60.png](https://user-images.githubusercontent.com/70456337/222933426-d8b3e597-44b7-493b-90e8-c0ecfee3e909.png)

![image-75.png](https://user-images.githubusercontent.com/70456337/222933427-0c914186-394c-453f-9b8d-06f04c5724de.png)

**Comment by mjbvz** [comment]
Please share example code (as text) that demonstrates the problem

**Comment by NightNutSky** [comment]
> Please share example code (as text) that demonstrates the problem

uh, it seems that something went wrong, so it was counted as an error. Probably VS Code restarting solved the issue (it's already another day and when I went sleep, i turned off my PC; therefore I already started my PC again).

I'll share that part of the code if `vsce package` will return an error which I showed above.
P.S. At that time, `vsce package` has returned this error too.
